roles/postgresql/defaults/main.yml

```diff
@@ -41,6 +41,9 @@ postgres_packages:
   - postgresql-client-{{ postgres_version }}
   - postgresql-contrib-{{ postgres_version }}
 
+dnf_postgres_packages:
```

Comment:
Where do we use this var?

Reply:
it was your suggestion ;-) and it is used here https://github.com/pulibrary/princeton_ansible/pull/5606/files/872428d142dcd542225d41e453e7c9a79d52125d#diff-68e937462c82c14f5ad880a4c3a8f2719e4a288d75e0a22641b524102414576bR47